genomic_data_service/region_indexer_encode.py

```diff
@@ -0,0 +1,140 @@
+    'field=uuid',
+    'field=assay_term_name',
+    'field=annotation_type',
+    'field=reference_type',
+    'field=collection_type',
+    'files.preferred_default=True',
+    '&'.join([f'status={st}' for st in ALLOWED_STATUSES]),
+    '&'.join([f'assembly={sa}' for sa in SUPPORTED_ASSEMBLIES]),
+    'format=json',
+    'limit=all'
+]
+
+
+def index_encode_files(query, requirements, ignore, dry_run=False):
+    endpoint = f"{SEARCH_ENDPOINT}?{'&'.join(query)}"
+
+    items = requests.get(endpoint).json()['@graph']
+
+    accessions = []
+
+    for item in items:
+        files = item.get('files', [])
+
+        for f in files:
+            if f['accession'] in ignore:
+                continue
+
+            valid = True
+            for requirement in requirements.keys():
+                if f.get(requirement, '') not in requirements[requirement]:
+                    valid = False
+
+            if valid:
+                accessions.append(f['accession'])
+
+                if not dry_run:
+                    index_file.delay(f, item, es_uri, es_port)
+
+    return accessions
+
+
+def encode_annotations(ignore=[], dry_run=False):
+    requirements = {
+        'file_format': ['bed']
+    }
+
+    query = [
+        'type=Annotation',
+        'encyclopedia_version=ENCODE+v5',
+        'annotation_type=representative+DNase+hypersensitivity+sites'
+    ] + ENCODE_QUERY
+
+    accessions = index_encode_files(query, requirements, ignore, dry_run)
+
+    query = [
+        'type=Annotation',
+        'encyclopedia_version=ENCODE+v5',
+        'annotation_type=candidate+Cis-Regulatory+Elements',
+        'annotation_subtype=all'
+    ] + ENCODE_QUERY
+
+    accessions_cis = index_encode_files(query, requirements, accessions + ignore, dry_run)
+
+    return accessions + accessions_cis
+
+
+def encode_functional_char_experiments(ignore=[], dry_run=False):
+    requirements = {
+        'file_format': ['bed'],
+        'file_type': ['bed bed6+'],
+        'preferred_default': [True]
+    }
+
+    query = [
+        'type=FunctionalCharacterizationExperiment',
+        'control_type!=*',
+        'audit.WARNING.category!=lacking+processed+data',
+        'files.file_type=bed+bed6%2B',
+    ] + ENCODE_QUERY
+
+    return index_encode_files(query, requirements, ignore, dry_run)
+
+
+def encode_experiments(ignore=[], dry_run=False):
+    requirements = {
+        'file_format': ['bed'],
+        'preferred_default': [True]
+    }
+
+    query = [
+        'type=Experiment',
+        '&'.join([f'assay_title={at}' for at in ASSAY_TITLES]),
+        'files.file_format=bed'
+    ] + ENCODE_QUERY
+
+    return index_encode_files(query, requirements, ignore, dry_run)    
+
+
+def index_encode_region_search(dry_run=False):
+    func_exps = encode_functional_char_experiments(dry_run=dry_run)
+    annotations = encode_annotations(ignore=func_exps, dry_run=dry_run)
+    return encode_experiments(ignore=(func_exps+annotations), dry_run=dry_run)
```

Comment:
`func_exps+annotations` => `func_exps + annotations`. Don't think the parentheses are necessary.